pydecred/account.py

```diff
@@ -225,15 +230,29 @@ def votingAddress(self):
             AddressSecpPubkey: The address object.
         """
         return AddressSecpPubKey(self.votingKey().pub.serializeCompressed(), self.net).string()
+    def addPool(self, pool):
+        """
+        Add the specified pool to the list of stakepools we can use.
+
+        Args:
+            pool (vsp.VotingServiceProvider): The stake pool object.
+        """
+        assert isinstance(pool, VotingServiceProvider)
+        # If this a new pool, give it an ID one more than the highest.
+        if pool.ID < 0:
+            pool.ID = 0
+            if len(self.stakePools) > 0:
+                pool.ID = max([p.ID for p in self.stakePools]) + 1
+        self.stakePools = [pool] + [p for p in self.stakePools if p.ID !=
+                                    pool.ID]
```

Comment:
Separated add and set because accountless should have different data to set each time we buy a ticket whenever we start to derive a new address for every ticket.